heron/cli/src/python/cli_helper.py

```diff
@@ -72,7 +72,7 @@ def run(command, parser, cl_args, unknown_args, action):
         "--command", command,
     ]
 
-    if opts.verbose():
+    if Log.getEffectiveLevel() == logging.DEBUG:
```

Comment:
Right. We compare to see if the logger object `Log` has the `DEBUG` level set. If it is set, we add ``--verbose`` flag to invoke Java program.